exp/zapslog/doc.go

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2023 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+// Package zapslog provides an implementation of slog.Handler which writes to
+// the supplied zapcore.Core.
+//
+// Since the slog proposal has not been officially accepted by the creation of this package,
+// we do not want Zap's standard packages to take a dependency on `golang.org/x/exp/slog`.
+// Instead, we provide this separate module as a way for users to integrate Zap with slog.
+package zapslog // import "go.uber.org/zap/exp/slog"
```

Comment:
nit: Why doesn't this import match the package name? If we want this import path, we should rename the package (and directory), else we should use the package name.

Reply:
Oh nice catch! I will fix this typo.